working/augmentation-libraries/feature-specification.md

```diff
@@ -115,7 +114,7 @@ library. Augmentations are distinguished by a special `library` directive with
 an `augment` modifier, like so:
 
 ```dart
-library augment 'main_library.dart';
+augment library 'main_library.dart';
```

Comment:
For what it's worth, my thinking with the original syntax was that you would read `library augment 'foo.dart'` as "a `library` that `augment`s `'foo.dart'`".

But I like the syntax you have here much better. I like that it parallels `augment` as a modifier on other declarations where `augment` precedes the thing it augments.